Gems/RecastNavigation/Code/Source/Misc/RecastNavigationPhysXProviderCommon.cpp

```diff
@@ -0,0 +1,232 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Misc/RecastNavigationPhysXProviderCommon.h>
+#include <AzCore/Console/Console.h>
+#include <AzCore/Debug/Profiler.h>
+#include <AzFramework/Physics/PhysicsScene.h>
+#include <AzFramework/Physics/Shape.h>
+#include <AzFramework/Physics/ShapeConfiguration.h>
+#include <DebugDraw/DebugDrawBus.h>
+
+AZ_CVAR(
+    bool, cl_navmesh_showInputData, false, nullptr, AZ::ConsoleFunctorFlags::Null,
+    "If enabled, draws triangle mesh input data that was used for the navigation mesh calculation");
+AZ_CVAR(
+    float, cl_navmesh_showInputDataSeconds, 30.f, nullptr, AZ::ConsoleFunctorFlags::Null,
```

Comment:
nit: I believe we prefer using `ed_` for editor CVAR settings now

Reply:
It is not just the Editor, these also apply to game mode.

Reply:
Ah okay got you thanks 👍